### ISSUE TITLE
Flutter 3.0 build error

### ISSUE THREAD

**Issue by zzterrozz** [Limbou/expandable_page_view#35]
After upgrading to Flutter 3.0 we got a build problem when using [expandable_page_view](https://github.com/Limbou/expandable_page_view).

Log:

```
: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
../…/src/size_reporting_widget.dart:23
- 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../../development/flutter/packages/flutter/lib/src/widgets/binding.dart').
package:flutter/…/widgets/binding.dart:1
    WidgetsBinding.instance!.addPostFrameCallback((_) => _notifySize());
                   ^
: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
```

**Comment by Limbou** [comment]
@zzterrozz This is just a warning, it should not prevent you from building the app. However I will upgrade and fix this warning with the upcoming version.

**Comment by Limbou** [comment]
Warnings fixes in #36, changes available in version 1.0.14
